CONTRIBUTING.md

```diff
@@ -234,3 +234,39 @@ run the following command and follow the prompts:
 ```
 yarn plop
 ```
+
+### Adding icons to Kaizen
```

Comment:
This doc is already pretty long. I suggest making a separate file and linking to it from here :)

Reply:
Agreed ^